src/g/zoomer.js

```diff
@@ -170,7 +170,8 @@ module.exports = Zoomer = Model.extend({
     }
 
     //@set "alignmentWidth", val
-    this.set("alignmentWidth", val)
+    // Note(ritik): Added 1px to fix the border getting clipped for last residue selection in sequences
+    this.set("alignmentWidth", val + 1)
```

Comment:
For some reason, the border for the last residue in sequences was being clipped.
![image](https://github.com/user-attachments/assets/b3626b3f-2b64-41a5-b6a8-b1fceeefe092)


Added one extra pixel to the width to fix the same.
![image](https://github.com/user-attachments/assets/f12e66c8-058c-40b0-9d92-ff7b43c94613)